HOWTO.md

```diff
@@ -5,37 +5,35 @@ This How-To will explain how to write a basic **Matrix <--> Slack bridge** in un
 
 You need to have:
  - A working homeserver install
- - `npm` and `nodejs`
+ - `npm` and Node.js
 
-NB: This how-to refers to the binary `node` - this may be `nodejs` depending on your distro.
+Note, this how-to refers to the binary of Node.js as `node` - on some Linux distros this may be called `nodejs`.
 
-# Setup a new project
+# Set up a new project
 Create a new directory and run `npm init` to generate a `package.json` file after answering some questions.
-Run `npm install matrix-appservice-bridge` to install the bridge library, `request` to make sending HTTP
-requests easier and `matrix-appservice` to install the AS library. Create a file `index.js` which we'll
-use to write logic for the bridge.
+Install bridge library `matrix-appservice-bridge` and `request` to make sending HTTP
+requests easier. Create a file `index.js` which we'll use to write the logic for the bridge.
 ```
 $ npm init
-$ npm install matrix-appservice-bridge
-$ npm install request
+$ npm install matrix-appservice-bridge request
 $ touch index.js
 ```
 
 # Slack-to-Matrix
 First, we need to create an Outgoing WebHook in Slack (via the Integrations section). This will send
-HTTP requests to us whenever a Slack user sends something in a slack channel. We'll monitor the channel
+HTTP requests to us whenever a Slack user sends something in a Slack channel. We'll monitor the channel
 `#matrix` when sending outgoing webhooks rather than trigger words. Set the URL to a publically accessible
 endpoint for your machine, or use something like [ngrok](https://ngrok.com/) if you're developing. We'll use
 ngrok, and forward port `$PORT`.
 
 Variables to remember:
  - Your monitored channel `$SLACK_CHAN`.
  
-## Printing out outbound slack requests
+## Printing out outbound Slack requests
 Open up `index.js` and write the following:
 ```javascript
-const http = require("http");
-const qs = require("querystring"); // we will use this later
+const http = require("node:http");

```

Comment:
TIL this is actually a feature in Node 14+. I thought this was new! Will have to remember to use it in my code :)